mdoc/Mono.Documentation/Updater/Statistics/StatisticsFormatter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Text;
+
+namespace Mono.Documentation.Updater.Statistics
+{
+    public static class StatisticsFormatter
+    {
+        public static string Format(Dictionary<string, StatisticsStorage> input)
+        {
+            var result = new StringBuilder();
+            foreach (var statisticsStoragePair in input)
+            {
+                var framework = statisticsStoragePair.Key;
+                var statisticsStorage = statisticsStoragePair.Value;
+                result.AppendLine(framework);
+                result.AppendLine("--------");
+                foreach (var statistics in statisticsStorage.Values)
+                {
+                    var staticsItem = statistics.Key;
+                    foreach (var statisticsValuePair in statistics.Value)
+                    {
+                        result.AppendLine(staticsItem + " " + statisticsValuePair.Key + ": " + statisticsValuePair.Value);
```

Comment:
Minor thing, but I'd like to see the use of string interpolation here:

```
result.AppendLine($"{staticsItem} {statisticsValuePair.Key}: {statisticsValuePair.Value}");
```

It makes the format of the line easier to read :)